lte/gateway/deploy/roles/magma_deploy/tasks/main.yml

```diff
@@ -55,52 +55,32 @@
     - agwc
     - base
 
-- name: Configuring private package manager.
-  copy:
-    content: >
-      deb {{ magma_pkgrepo_proto }}://{{ magma_pkgrepo_host }}{{ magma_pkgrepo_path }}
-      {{ magma_pkgrepo_dist }} {{ magma_pkgrepo_component }}
-    dest: /etc/apt/sources.list.d/{{ magma_pkgrepo_name }}.list
+- name: Add key for magma registry
+  ansible.builtin.copy:
+    src: "{{ magma_pkgrepo_key }}"
+    dest: /etc/apt/trusted.gpg.d/magma.key
   become: true
   tags:
     - agwc
     - base
 
-- name: Add unvalidated Apt signing key.
-  when: magma_pkgrepo_key_fingerprint == ""
-  become: true
-  block:
-    - name: Download key
-      apt_key:
-        url: "{{ magma_pkgrepo_key }}"
-        state: present
-        validate_certs: no
-    - name: Ignore server ssl cert
-      copy:
-        dest: /etc/apt/apt.conf.d/99insecurehttpsrepo
-        content: |
-          Acquire::https::{{ magma_pkgrepo_host }} {
-          Verify-Peer "false";
-          Verify-Host "false";
-          };
+- name: Ignore server ssl cert
```

Comment:
Have you checked if we actually need this?

Reply:
That is a good question. Given that this basically is an exact copy of the pkgrepo Ansible step it can probably removed. But I didn't want to risk anything and didn't check it.

Reply:
I would hope that we can actually remove it both here and in pkgrepo. But maybe that's something for a another PR.

Reply:
#14629